website/templates/base.mako

```diff
@@ -173,6 +173,15 @@
             </script>
         % endif
 
+        %if keen_project_id:
+            <script>
+                window.contextVars = $.extend(true, {}, window.contextVars, {
+                    keenProjectId: '${keen_project_id}',
+                    keenWriteKey: '${keen_write_key}'
```

Comment:
The plan for Keen is to do something a la Tornadik with some munger/server middle man that takes all requests which holds the proper Keen settings. However, until that is implemented, there is no way to send data to Keen through js without exposing our keys. If you have a suggestion as to what should be done, I would gladly take the help.